hudi-client/hudi-flink-client/src/main/java/org/apache/hudi/client/model/HoodieFlinkRecord.java

```diff
@@ -216,7 +217,7 @@ public HoodieRecord rewriteRecordWithNewSchema(Schema recordSchema, Properties p
   }
 
   @Override
-  protected boolean checkIsDelete(Schema recordSchema, Properties props) {
+  protected boolean checkIsDelete(DeleteContext deleteContext, Properties props) {
```

Comment:
@danny0405 I could not find any way to import the `FlinkRecordContext` in this module. How should this record type be updated to make sure it is consistent with the other paths?